adserver/utils.py

```diff
@@ -31,13 +31,23 @@
 log = logging.getLogger(__name__)  # noqa
 
 
-GeolocationTuple = namedtuple(
-    "GeolocationTuple", ["country_code", "region_code", "metro_code"]
-)
-
-
 # Put this here so we don't reload it on each call
 COUNTRY_DICT = dict(countries)
+COUNTRY_DICT["T1"] = "Tor"
```

Comment:
I think this probably won't be included in any of our aggregate reports currently, if it isn't in the `regiontopics` listing. Probably fine though..

Reply:
I don't think we ever want it anywhere. I am just noting it for completeness.